app/components/moderation-list-row/component.js

```diff
@@ -13,22 +13,42 @@ const ACTION_LABELS = Object.freeze({
         ltDay: 'components.moderation-list-row.submission.submitted',
     },
     [ACCEPTED]: {
-        gtDay: 'components.moderation-list-row.submission.was_accepted_on',
-        ltDay: 'components.moderation-list-row.submission.was_accepted',
+        gtDay: 'components.moderation-list-row.submission.accepted_on',
+        ltDay: 'components.moderation-list-row.submission.accepted',
+        gtDay_automatic: 'components.moderation-list-row.submission.accepted_automatically_on',
+        ltDay_automatic: 'components.moderation-list-row.submission.accepted_automatically',
     },
     [REJECTED]: {
-        gtDay: 'components.moderation-list-row.submission.was_rejected_on',
-        ltDay: 'components.moderation-list-row.submission.was_rejected',
+        gtDay: 'components.moderation-list-row.submission.rejected_on',
+        ltDay: 'components.moderation-list-row.submission.rejected',
     },
 });
 
 
 export default Component.extend({
     theme: service(),
+    i18n: service(),
+
+    noActions: false,
 
     classNames: ['moderation-list-row'],
 
-    dataLoading: computed.not('firstContributors.length'),
+    contributorLoading: computed.not('firstContributors.length'),
+
+    moderatorLoading: computed.none('lastActionCreator'),
+
+    lastActionCreator: computed.alias('latestAction.creator.fullName'),
```

Comment:
Please be consistent with names; choose either `latest` or `last` and stick with it